tempora/timing.py

```diff
@@ -51,16 +51,21 @@ def reset(self):
             del self.start_time
 
     def start(self):
-        self.start_time = datetime.datetime.utcnow()
+        self.start_time = datetime.datetime.now(datetime.timezone.utc).replace(
+            tzinfo=None
+        )
```

Comment:
This implementation retains maximum backwards compatibility. It does not meet the gist of the deprecation where it is recommended to use aware datetimes to represent UTC though.

 If `start_time` and `end_time` are not considered part of the API, I would personally convert the implementation to use `time.monotonic()` (or `..._ns()`) for them to avoid breakage when the system clock updates, e.g. at DST changes.

Alternatively, switching them to use aware datetimes would in my opinion be an improvement with some, but less drastic API change.

Reply:
I agree, `monotonic` is probably preferable here. This code predates the introduction of monotonic, but I'd like to use it. Let's do the datetime-aware object for now and convert to `monotonic` in a separate PR. Thanks!

Reply:
Done.